src/taskgraph/util/schema.py

```diff
@@ -157,6 +157,7 @@ def resolve_keyed_by(
     # which expects interCaps
     "upstream-artifacts",
     "artifact-map",
+    lambda item: "schema['notify']['context']" in item,
```

Comment:
Bikeshed: I wonder if we can avoid the camelCase here because the generated task should be just contain some routes.